src/definitions/index.ts

```diff
@@ -137,4 +137,4 @@ const TransactionResult = new BytesLookup(
 );
 const Field = new FieldLookup(enums.FIELDS as Array<[string, FieldInfo]>);
 
-export { Field, Type, LedgerEntryType, TransactionResult, TransactionType };
+export { Field, FieldInstance, Type, LedgerEntryType, TransactionResult, TransactionType };
```

Comment:
Also, not married to the name `FieldInstance` on for the `interface FieldInstance {...}`. Open to suggestions for a better name.